### PR TITLE
Update cookbook to Chef 15

### PR DESCRIPTION
**Pull request by erickuiper**

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -1,28 +1,27 @@
...
+  archive_file zipfile do
+    path temppath
+    destination node['sysinternals']['install_dir']
+    action :extract
+  end
 end
 
-if !node['sysinternals']['bginfo_config_url'].nil? && !node['sysinternals']['bginfo_config_dir'].nil?
-  include_recipe 'sbp_sysinternals::bginfo'
+unless node['sysinternals']['bginfo_config_url'].nil? || node['sysinternals']['bginfo_config_dir'].nil?
```
Comment (mvangoor) [inline recipes/default.rb]: Please rewrite this to:
`if node['sysinternals'].key?('bginfo_config_url') && node['sysinternals'].key?('bginfo_config_dir')`

File: attributes/default.rb [hunk 2]
```diff
@@ -1,23 +1,3 @@
...
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-default['sysinternals']['url']               = 'https://download.sysinternals.com/files/SysinternalsSuite.zip'
-default['sysinternals']['install_dir']       = "#{ENV['SYSTEMDRIVE']}\\SysinternalsSuite"
-default['sysinternals']['bginfo_config_url'] = nil
```
Comment (mvangoor) [inline attributes/default.rb:22]: Please leave the bginfo* options in there, but commented with a remark they need to be set if they want to use it

File: recipes/default.rb (hunk 1 above)
Comment (erickuiper) [reply recipes/default.rb]: Has been updated

File: recipes/default.rb [hunk 3]
```diff
@@ -1,28 +1,27 @@
...
+    path temppath
+    destination node['sysinternals']['install_dir']
+    action :extract
+  end
 end
 
-if !node['sysinternals']['bginfo_config_url'].nil? && !node['sysinternals']['bginfo_config_dir'].nil?
-  include_recipe 'sbp_sysinternals::bginfo'
+if node['sysinternals'].key?('bginfo_config_url') && node['sysinternals'].key?('bginfo_config_dir')
+  include_recipe "#{cookbook_name}::bginfo"
```
Comment (mvangoor) [inline recipes/default.rb]: Please update this to `include_recipe '::bginfo'`

**Review by mvangoor — APPROVED** [review]

**Pull request merged**